packages/services/questionnaire/src/lib/inputs/input.svelte

```diff
@@ -67,15 +70,31 @@
         color: greenyellow;
     }
 
+    .question-description {
+        width: 100%;
+        text-align: justify;
+    }
+
     .hub-input {
         width: 500px;
         height: 100px;
+        margin-bottom: 50px;
     }
-
-    @media only screen and (max-device-width: 800px) {
+    @media only screen and (max-width: 800px) {
         .hub-input {
-            width: 300px;
+            width: 475px;
+        }
+    }
+    @media only screen and (max-width: 700px) {
```

Comment:
I believe we can remove this using https://tailwindcss.com/docs/responsive-design as well